### DISCUSSION TITLE
Instances in yellow since yesterday

### DISCUSSION THREAD

**Discussion by dedalo88888** [KevinBytesTheDust/CVAmp#236] [Q&A] [answered]
Since 03/01/2024 I have had some instances that turn yellow and others turn blue.
![20240105_092841_instance2](https://github.com/jlplenio/crude-twitch-viewer-bot/assets/44909583/536e8f45-8d69-4370-a727-b60d57dd4905)
What I can do?

**Comment by dedalo88888** [comment, marked as answer]
After checking I realized that it is because Twitch banned the webshare.io IP

**Reply by gShazaa** [reply]
So all proxies from webshare are down?

**Reply by dedalo88888** [reply]
Not all of them, but Twitch did a sweep and some do not work with Tocken but they do work without Tocken or so it seems

**Reply by jlplenio** [reply]
If you replace the proxies with new ones in webshare, does it work?

**Reply by dedalo88888** [reply]
Correct, when replacing the proxies all the instances with the tokens worked again.
The curious thing is that from one day to the next +60 instances in yellow and the rest in blue
